libafl/src/executors/forkserver.rs

```diff
@@ -740,6 +811,15 @@ where
             self.map_input_to_shmem(input, input_size)?;
         }
 
+        // clear observers
+        if let Some((stdout, _)) = self.forkserver.stdout_memfd.as_mut() {
```

Comment:
these kinds of things are usually done in observers.pre_exec()
for example, we cleanup the map observer in pre_exec().. etc
and putting this code in executor is not good.
any chance that you can move all this as an observer?

Reply:
This makes sense. We should have a wrapper for those observers.